setup.py

```diff
@@ -33,13 +45,7 @@
     namespace_packages=['collective'],
     include_package_data=True,
     zip_safe=False,
-    install_requires=[
-        'setuptools',
-        'simplejson',
-        'collective.transmogrifier',
```

Comment:
I don't know if we need collective.transmogrifier, I have not studied the code. If we are importing it or otherwise directly using it, we should have it in the `install_requires` list.

Imagine that `plone.app.transmogrifier` is someday refactored and loses its dependency on `collective.transmogrifier`. Would we then need this dependency after all?

Note: I have used collective.jsonmigrator only once in a test, so I hardly know what it is doing. :-) I am simply looking at a pull request that claims to handle json/simplejson better and that almost unnoticed removes a dependency which has got nothing to do with json. So then I start wondering if this is a simple copy-paste error. :-)